guides/common/modules/snip_host-names-for-http-proxy.adoc

```diff
@@ -12,3 +12,7 @@ ifdef::satellite[]
 | cert-api.access.redhat.com (if using Red{nbsp}Hat Insights) | 443 | HTTPS
 endif::[]
 |====
+
+ifdef::orcharhino[]
```

Comment:
This diff should not be part of your PR. Can you please rebase? Then I'll merge.